release-doc-in-process/all_vf.TTL

```diff
@@ -265,17 +265,29 @@ vf:appreciationWith
         rdfs:range          vf:EconomicEvent ;
         rdfs:comment        "The economic event implemented in appreciation (gift economy)." .
 
-vf:provider  a       owl:ObjectProperty ;
-        rdfs:domain  [ owl:unionOf (vf:EconomicEvent vf:Commitment) ] ;
-        rdfs:label   "provider" ;
+vf:commitedBy  a     owl:ObjectProperty ;
+        rdfs:domain  vf:Commitment ;
+        rdfs:label   "commitedBy" ;
+        rdfs:range   foaf:Agent ;
+        rdfs:comment "The economic agent who makes the commitment." .
+
+vf:performedBy  a    owl:ObjectProperty ;
+        rdfs:domain  vf:EconomicEvent ;
+        rdfs:label   "performedBy" ;
```

Comment:
each event has *vf:action*, one of the available values *vf:cite*, i think this action gets performed or executed or carriet just done. i would like to move away from *provide* since for example for events with `"vf:action": "vf:consume"` having agent who carries that action as *providedBy* seems a little misleading.